adapters/kobler/kobler.go

```diff
@@ -0,0 +1,143 @@
+package kobler
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/prebid/openrtb/v20/openrtb2"
+	"github.com/prebid/prebid-server/v2/adapters"
+	"github.com/prebid/prebid-server/v2/config"
+	"github.com/prebid/prebid-server/v2/errortypes"
+	"github.com/prebid/prebid-server/v2/openrtb_ext"
+)
+
+type adapter struct {
+	endpoint    string
+	devEndpoint string
+}
+
+const (
+	DEV_BIDDER_ENDPOINT = "https://bid-service.dev.essrtb.com/bid/prebid_server_rtb_call"
+	SUPPORTED_CURRENCY  = "USD"
+)
+
+func Builder(bidderName openrtb_ext.BidderName, config config.Adapter, server config.Server) (adapters.Bidder, error) {
+	bidder := &adapter{
+		endpoint:    config.Endpoint,
+		devEndpoint: DEV_BIDDER_ENDPOINT,
+	}
+
+	return bidder, nil
+}
+
+func (a adapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.ExtraRequestInfo) (requestData []*adapters.RequestData, errors []error) {
```

Comment:
var requestData []*adapters.RequestData
var errs []error
Initializing requestData and errs at the beginning of the function helps avoid implicit null values when returning an empty list (null pointer). It also makes the function more readable, as it immediately shows which variables are used for storing results and errors.